state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1210,6 +1400,8 @@ pub mod pallet {
 			destination_address: ForeignChainAddress,
 			deposit_metadata: CcmDepositMetadata,
 			origin: SwapOrigin,
+			// TODO: CCM should use refund params
+			_refund_params: Option<ChannelRefundParameters>,
```

Comment:
I wouldn't. This is exactly the code that I will need for CCM which I will likely start writing today/tomorrow. Removing this just to add it later doesn't benefit anyone.

Reply:
It's about PRs being a single unit of work. If we decide to exclude CCM from this PR, then let's exclude it from this PR. We're just leaking things in which aren't necessary until CCM is added. We can't be sure we'll add it right after, there's a lot to do, and in the meantime, there's this dangling code that doesn't do anything.

Reply:
> It's about PRs being a single unit of work. 

I agree that it is a good idea in general (and I probalbly wouldn't add any ccm related code if I knew that ccm would turn out to be more difficult and would have to go in a separate PR), but there is value in being flexible too. General principles like this are important, but it is more important to understand why we follow them, and be flexible enough to make exceptions when the "why" does not really apply.

Reply:
Not gonna block the PR because of it, but the "why" here always applies. A single unit of work is a single unit of work. It's clearly leaking details of another PR in here. Makes it harder to track where changes are introduced etc. too when tracing history. Easier when a single unit is just a single PR.